Godeps/_workspace/src/k8s.io/kubernetes/pkg/admission/plugins.go

```diff
@@ -36,8 +38,29 @@ type Factory func(client clientset.Interface, config io.Reader) (Interface, erro
 var (
 	pluginsMutex sync.Mutex
 	plugins      = make(map[string]Factory)
+
+	defaultOnPlugins  = sets.String{}
+	defaultOffPlugins = sets.String{}
+
+	PluginEnabledFn = func(config io.Reader, defaultValue bool) (bool, error) {
```

Comment:
add a comment describing that the default behavior is that all plugins are enabled if they are registered in code.

note that api servers can restrict the set of enabled plug-ins by providing their own implementation of the function.